contrib/build-push-ceph-container-imgs.sh

```diff
@@ -248,13 +252,13 @@ function push_ceph_imgs_latest {
     local_tag=${CONTAINER_REPO_ORGANIZATION}/daemon-base:${RELEASE}-${BRANCH}-centos-${CENTOS_RELEASE}-${HOST_ARCH}
     full_repo_tag=${CONTAINER_REPO_HOSTNAME}/${CONTAINER_REPO_ORGANIZATION}/ceph:${RELEASE}-centos-${CENTOS_RELEASE}-${HOST_ARCH}-devel
     branch_repo_tag=${CONTAINER_REPO_HOSTNAME}/${CONTAINER_REPO_ORGANIZATION}/ceph:${BRANCH}
-    sha1_repo_tag=${CONTAINER_REPO_HOSTNAME}/${CONTAINER_REPO_ORGANIZATION}/ceph:${SHA1}
+    sha1_os_flavor_repo_tag=${CONTAINER_REPO_HOSTNAME}/${CONTAINER_REPO_ORGANIZATION}/ceph:${SHA1}-${OS}-${CENTOS_RELEASE}-${OSD_FLAVOR}
     docker tag $local_tag $full_repo_tag
     docker tag $local_tag $branch_repo_tag
     docker tag $local_tag $sha1_repo_tag
     docker push $full_repo_tag
     docker push $branch_repo_tag
-    docker push $sha1_repo_tag
+    docker push $sha1_os_flavor_repo_tag
```

Comment:
i don't think this works. you need to fix the previous command of 
```sh
docker tag $local_tag $sha1_repo_tag
```
first. namely to create a tag of `$sha1_os_flavor_repo_tag` referring the `$local_tag`. also, we are still overwriting the`$full_repo_tag` and `$branch_repo_tag` with the last flavor pushed to registry.

i am not sure who are the consumers of the other two tags, but if there are any of them. they will likely have random failures. as crimson-osd will be used in the place of ceph-osd, if the crimson flavored image is pushed later.

Reply:
aah, sorry for missing that. Even I am not sure about consumer for them but I don't think that will cause failure, as we only supply OSD_FLAVOR to push-container-img.sh script in Jenkins, and apart from that OSD_FLAVOR defaults to use `default` flavor, so I don't suspect it to fail.

Reply:
I'm not aware about the consumers, this has been design by @dmick for the ceph CI so it might have more information